source/getting-started/index.markdown

```diff
@@ -25,43 +45,35 @@ We will need a few things to get started with installing Home Assistant. For bes
 - [Micro SD Card](http://a.co/gslOydD). Get one that is Class 10 as they are more reliable. Size 32 GB or bigger recommended.
 - SD Card reader. Part of most laptops, and also available as [standalone USB sticks](http://a.co/5FCyb0N) (the brand doesn't matter, just pick the cheapest)
 - Ethernet cable (optional, Hass.io can work with WiFi as well)
-- A USB-Stick (optional, allows for unattended configuration)
 
 ### {% linkable_title Software requirements %}
 
 - Download the Hass.io image for [your device](/hassio/installation/)
 - Download [balenaEtcher] to write the image to an SD card
-- Text Editor like [Visual Studio Code](https://code.visualstudio.com/)
 
 [balenaEtcher]: https://www.balena.io/etcher
 
 ### {% linkable_title Installing Hass.io %}
 
 1. Put the SD card in your SD card reader.
 1. Open balenaEtcher, select the Hass.io image and flash it to the SD card.
-1. WiFi and Static IP setup only: 
+1. Unmount the SD card and remove it from your SD card reader.
+1. Only if you want to configure WiFi or a Static IP (requires USB stick):
    - Format a USB-Stick to FAT32 with volume-name `CONFIG`.
    - Create a folder named `network` in the root of the newly formatted USB-stick.
    - Within that folder create a file named `my-network` without extension.
-   - Copy one of [the examples] to the `my-network` file.
+   - Copy one of [the examples] to the `my-network` file and adjust accordingly.
    - Plug the USB-stick into the Raspberry Pi 3.
-1. Unmount the SD card and remove it from your SD card reader.
+
 1. Insert the SD card into your Raspberry Pi 3. If you are going to use an Ethernet cable, connect that too.
 1. Connect your Raspberry Pi to the power supply, so it turns on.
 1. The Raspberry Pi will now boot up, connect to the Internet and download the latest version of Home Assistant, which will take about 20 minutes.
-1. Home Assistant will be available at [http://hassio.local:8123][local]. Shortly after the download has started, a simple preparation status page will be available at this URL. Point your browser there. The page refreshes automatically, and Home Assistant will be shown when the download is complete (this typically takes around 20 minutes).
-1. Please remember to [secure your installation][secure] once you've finished with the installation process.
+1. Home Assistant will be available at [http://hassio:8123][host] if you're on Windows, otherwise at [http://hassio.local:8123][local].
```

Comment:
Instead of _you're on Windows_ maybe _using Windows to access it_

This _might_ be misread as being if you install Hass.io on Windows